scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -20,15 +20,28 @@ class SensorVersionsFailure(Exception):
 
 gkecluster=GKECluster("compat-test")
 
+if len(chart_versions) > 1:
+    os.environ["CENTRAL_CHART_VERSION"] = chart_versions[0]
+
 failing_sensor_versions = []
 for version in chart_versions:
     os.environ["SENSOR_CHART_VERSION"] = version
     try:
         make_compatibility_test_runner(cluster=gkecluster).run()
     except Exception:
-        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version}")
+        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version} and central version {chart_versions[0]}")
         failing_sensor_versions += version
 
+if len(chart_versions) > 1:
+    os.environ["SENSOR_CHART_VERSION"] = chart_versions[0]
+    os.environ["CENTRAL_CHART_VERSION"] = chart_versions[1]
+    try:
+        make_compatibility_test_runner(cluster=gkecluster).run()
+    except Exception:
+        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {chart_versions[0]} and central version {chart_versions[1]}")
+        failing_sensor_versions += version
+
+
 if len(failing_sensor_versions) > 0:
-    raise SensorVersionsFailure(f"Compatibility tests failed for Sensor versions " + ', '.join(failing_sensor_versions))
+    raise SensorVersionsFailure(f"Compatibility tests failed for Sensor versions " + ', '.join(failing_sensor_versions) + " and Central versions " + ', '.join(failing_sensor_versions[:2]))
```

Comment:
Is there no mistake about what gets into "Central versions"?

Again, if you adopt the idea with "list of tuples/structs to test", you can simply append failing tuples/structs to the "failing list" and in the end print out this list without the need of much hand-written formatting.

Reply:
Done